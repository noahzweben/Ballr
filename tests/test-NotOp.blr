gameboard g1{
	clr = (251,142,74);
	size = (680,420);
	init -> {

		int a= 5; 
    int b= 1; 

    while (a!=b){

      print (a); 

      a= a-1; 
    }
}