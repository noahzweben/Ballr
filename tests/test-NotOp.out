5
4
3
2
SDL could not initialize! SDL Error: No available video device
Window initialization failed.